### ISSUE TITLE
Is 'validation_split' implemented

### ISSUE THREAD

**Issue by amjass12** [keras-team/keras-tuner#267]
Hi all, 

I am just about to run a HP for a couple models i am building. I just want to confirm as I haven't found this anywhere, either on the tutorial page or other documentation present if the'validation_split' argument is present in the tuner.search function for fitting a model? or if X_val and y_val explicitly have to be implemented?

the code for tuner.search I am using is as follows::

```

tuner.search(X_train.values, y_train,
             epochs=200,
             validation_split=0.1,
             callbacks=[EarlyStopping(monitor='val_loss', patience=5)], shuffle=True)

```
So the question is, does validation split automatically split the x and y_train data into a validation by 10%?

running this code suggests that it does indeed as validation loss and val_acc are present during training but just want to confirm as I havent see this documented anywere. 

many thanks!!

**Comment by ben-arnao** [comment]
Yes, by default any params you pass to search() are arguments passed to fit (assuming you don't override run_trial())

**Comment by amjass12** [comment]
Hi @ben-arnao ,

thank you so much for confirming! yes i have only built the def_model function, generated the random search (tuner=Randomsearch etc) and then tuner.search... nothing else. I assume this is the normal pipeline. 

I have more question, I have just received some extra labelling data now where i am able to build a  multi-task model (same input, multiple outputs).. is it possible to implement keras tuner for a multi-output model?

thanks!

**Comment by ben-arnao** [comment]
@amjass12 
I'm not sure exactly what is meant by a multi-output model, one where you will want to use multiple loss functions?

Either way, yes it is definitely possible even if you have to define your own hyperparameters -> model score logic. 

```
class MyTuner(kt.Tuner):

    def run_trial(self, trial, ...):
        model = self.hypermodel.build(trial.hyperparameters)
        score = ...
        self.oracle.update_trial(trial.trial_id, {'score': score})
        self.oracle.save_model(trail.trial_id, model)
```

At a basic level, in the run_trial method, you are supplied a set of parameters

`trial.hyperparameters`

and it is entirely up to you how you want to train your model with this params, all you need to do at the end is set the score.

`self.oracle.update_trial(trial.trial_id, {'score': score})`

However, if this is something you can do with the standard keras API then it will work in KT without needing to implement this yourself.

**Comment by vb690** [comment]
@amjass12

For multi-task learning Keras optimizes a weighted (or not) combination of each head's loss (i.e. `w0 * loss0 + w1 * loss1` in your case). 

This means that the tuner will receive from the model a single global loss and pick the best hyperparameters according to that.

Long story short, kerastuner behaves the same way whether the model has 1 or n heads.

**Comment by amjass12** [comment]
@vb690 

thank you! all this makes sense, i will proceed with keras tune ron my model now! thanks all again :)
